src/py/aspen/database/models/phylo_tree.py

```diff
@@ -1,13 +1,40 @@
-from sqlalchemy import Column, ForeignKey, Integer, String, UniqueConstraint
+from sqlalchemy import (
+    Column,
+    ForeignKey,
+    Integer,
+    String,
+    Table,
+    text,
+    UniqueConstraint,
+)
+from sqlalchemy.dialects.postgresql import JSONB
 from sqlalchemy.orm import backref, relationship
 
+from aspen.database.models.base import base
 from aspen.database.models.entity import Entity, EntityType
+from aspen.database.models.sample import Sample
 from aspen.database.models.usergroup import Group
 from aspen.database.models.workflow import Workflow, WorkflowType
 
+_PHYLO_TREE_TABLENAME = "phylo_trees"
+
+
+PhyloTreeSamples = Table(
+    "phylo_tree_samples",
+    base.metadata,
+    Column("sample_id", ForeignKey(Sample.id), primary_key=True),
+    Column(
+        "phylo_tree_id",
+        ForeignKey(f"{_PHYLO_TREE_TABLENAME}.entity_id"),
+        primary_key=True,
+    ),
+)
+"""This table records which samples are included in the phylo tree.  Note that this is
+different from which samples are inputs into the build that creates a phylo tree."""
```

Comment:
can we add this as a docstring to whatever model this is referring to?

Reply:
This is applicable to `PhyloTreeSamples`.  It is somewhat a quirk of python docs that the docstring appears below, even in the case of variables.